website/static/spec/OpenLineage.yml

```diff
@@ -1,7 +1,7 @@
 openapi: 3.0.2
 info:
   title: OpenLineage
-  version: 0.0.1
+  version: 2-0-2
```

Comment:
If this is not the must have, I would love to avoid this within this PR. 
It's kind of gargantuan to follow and I really like that it only modifies `integration/hive`, website and ci.